### DISCUSSION TITLE
Search enterprise only option

### DISCUSSION THREAD

**Discussion by holbrookab** [community/community#3851] [Enterprise]
Hi,

Love the mobile GH app.  What would make it better is a way to only search my company, versus all of GitHub.  This would be especially helpful when trying to find a particular repo.

Thanks!

**Comment by nplasterer** [comment]
Thanks for the feedback. We're looking into ways to improve search through out the app and will take this into account.

**Comment by Timmmy-nlb** [comment]
There are here some news?
